cpp/dolfin/generation/UnitDiscMesh.cpp

```diff
@@ -92,8 +93,11 @@ mesh::Mesh UnitDiscMesh::create(MPI_Comm comm, int n,
     }
   }
 
+  Eigen::Array<std::int64_t, Eigen::Dynamic, 6, Eigen::RowMajor> cells_reordered
```

Comment:
Same. Let's just change the code above to provide the correct ordering.

Reply:
This one is slightly more complicated to change, as the connectivity is made manually. I do not really want change these.

Reply:
Can we remove `UnitDiscMesh` now that we have straightforward support for (py)gmsh?

Reply:
Yes. We should probably also remove UnitTriangleMesh and UnitTetrahedronMesh